.isort.cfg

```diff
@@ -0,0 +1,8 @@
+[settings]
+force_to_top=file1.py,file2.py
+known_django=django
+include_trailing_comma=True
+line_length=88
+multi_line_output=3
+sections=FUTURE,STDLIB,DJANGO,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
+skip_glob=/app/node_modules/**/*
```

Comment:
Can you add this configuration to the `setup.cfg` file instead and remove this file?